scripts/add-token.js

```diff
@@ -3,36 +3,34 @@ import {
   erc20Abi,
   getAddress as toChecksum,
   http,
-  isAddress,
   isAddressEqual,
 } from "viem";
 import { hemi, hemiSepolia } from "hemi-viem";
 import fs from "node:fs";
 
-// eslint fails to parse "with { type: "json" }"
-// See https://github.com/eslint/eslint/discussions/15305
-const tokenList = JSON.parse(fs.readFileSync("./src/hemi.tokenlist.json"));
```

Comment:
Just to clarify:

This comment mentioned that using

```tsx
import tokenList from './src/hemi.tokenlist.json' with { type: "json" } 
```

works but throws an eslint error, and that's why we're using `fs`